packages/terra-base/src/Base.jsx

```diff
@@ -1,38 +1,74 @@
 import React, { PropTypes } from 'react';
-import './Base.scss';
+
+import { I18nProvider, i18nLoader } from 'terra-i18n';
+
+import BaseStyles from './BaseStyles';
 
 const propTypes = {
   /**
    * The component(s) that will be wrapped by `<Base />`.
    */
   children: PropTypes.node.isRequired,
+  /**
+   * Customized translations provided by consuming application.
+   */
+  customMessages: PropTypes.object,
+  /**
+   * The locale name.
+   */
+  locale: PropTypes.string,
+};
+
+const defaultProps = {
+  customMessages: {},
+  locale: 'en',
```

Comment:
I think we should remove the default locale.

If a user defines the locale as fi-FI on the server and it not configured to update on the client, then we can can multiple locales on the same page.